test/files/pos/t5075.scala

```diff
@@ -0,0 +1,38 @@
+//Copied from Scalaz - http://scalaz.googlecode.com/svn/continuous/latest/browse.sxr/scalaz/PartialApplys.scala.html
+trait PartialApply1Of2[T[_, _], A] {
+  type Apply[B] = T[A, B]
+
+  type Flip[B] = T[B, A]
```

Comment:
Was `Flip` to be used for something?

Reply:
Guess I just forgot to minimize it away in 2011. But it's useful for your comment https://github.com/scala/bug/issues/5075#issuecomment-370148938.

Reply:
@Blaisorblade I suspect that a modern minimization of 5075 would use `Flip` instead, no?

Reply:
Of course, I already upvoted your comment! (Nitpick: I'd add testcases with `Flip` in addition.)